src/save_options/options.py

```diff
@@ -15,11 +15,25 @@
     get_pc,
     hook,
 )
-from mods_base.options import DropdownOption, KeybindOption
+from mods_base.options import BaseOption, DropdownOption, GroupedOption, KeybindOption, NestedOption
 
 any_option_changed: bool = False
 
 
+def set_option_to_default(save_option: BaseOption) -> None:
+    """
+    Sets an option's value to its default.
+
+    For GroupedOption and NestedOption, recursively sets all children to their default values.
+    """
+    if isinstance(save_option, ValueOption):
```

Comment:
Nitpick: if you were to edit this code again I'd prefer using a match. Fine as is.

Reply:
Yeah seems we have to use it now...

Reply:
I suspect this comes from build 399
> Extended `reportUnknownVariableType` check to include cases where a value whose type is a generic class parameterized by Unknown is assigned to a variable with a declared type.